linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/H2Config.scala

```diff
@@ -177,11 +166,22 @@ class H2PrefixConfig(prefix: PathMatcher) extends PrefixConfig(prefix) with H2Cl
 
 trait H2ClientConfig extends ClientConfig with H2EndpointConfig {
   var forwardClientCert: Option[Boolean] = None
+  var requestAuthorizers: Option[Seq[H2RequestAuthorizerConfig]] = None
 
   @JsonIgnore
   override def params(vars: Map[String, String]): Stack.Params =
     withEndpointParams(super.params(vars))
       .maybeWith(forwardClientCert.map(ForwardClientCertFilter.Enabled))
+      .maybeWith(loggerParam)
+
+  @JsonIgnore
+  private[this] def loggerParam = requestAuthorizers.map { configs =>
```

Comment:
lets take this opportunity to rename this.  (they used to be called "loggers" before they were renamed to "requestAuthorizers")